lib/options.js

```diff
@@ -124,10 +124,7 @@ function assignPublishing(opts) {
 
   if (opts.ably && opts.channel) {
     try {
-      opts.ably = opts.ably === true ? {} : opts.ably || {};
-      opts.ably.log = { level: 0 };
-
-      var ably = new require('ably').Rest(opts.ably);
+      var ably = new require('ably').Rest({log: {level: 0}});
```

Comment:
Definitely make sure to at least set NPM package versions to `^x.y.z`. I'd also **highly** recommend just using [`npm shrinkwrap`.](https://docs.npmjs.com/cli/shrinkwrap).

There's really no reason to allow for different versions across different build environments. Best to lock everything in and make all package upgrades explicit.